cmd/assets/db_tables.txt

```diff
@@ -0,0 +1,134 @@
+census_duns21
```

Comment:
Ask Sudha if we can remove these tables entirely census_abc#

Reply:
TBD, but not a blocker for review.